src/resources/lang/es/crud.php

```diff
@@ -139,11 +160,21 @@
     'internal_link_placeholder' => 'Slug interno. Ejplo: \'admin/page\' (sin comillas) para \':url\'',
     'external_link'             => 'Enlace externo',
     'choose_file'               => 'Elegir archivo',
+    'new_item'                  => 'Nuevo elemento',
+    'select_entry'              => 'Selecciona un registro',
+    'select_entries'            => 'Selecciona registros',
 
     // Table field
     'table_cant_add'    => 'No se puede agregar una nueva :entity',
     'table_max_reached' => 'El número máximo de :max alcanzado',
 
     // File manager
     'file_manager' => 'Administrador de archivos',
+
+    // InlineCreateOperation
+    'related_entry_created_success' => 'Los registros relacionados se han creado correctamente.',
+    'related_entry_created_error' => 'No puedes crear registros relacionados.',
```

Comment:
Hola @karrikas!
Thank you for this addition 👌

One question, can you please make sure this translation is correct?

In this case, this message will appear when there is some kind of error on the back-end side. Not necessarily because the user can't create the entry, it may be other error.

```suggestion
    'related_entry_created_error' => 'No se puede crear los registros relacionados.',
```

I saw above there is a translation stating `No se puede agregar una nueva :entity` that makes more sense.

Therefor my suggestion, but, as a Portuguese I can only minimally understand Spanish, so as you are a native speaker please let me know if I'm wrong 😅

Thank you 🙌